src/app/core/user/user-list/user-list.component.ts

```diff
@@ -0,0 +1,57 @@
+import { Component, Input } from "@angular/core";
+import { User } from "../user";
+import { EntityMapperService } from "../../entity/entity-mapper.service";
+import { OnInitDynamicComponent } from "../../view/dynamic-components/on-init-dynamic-component.interface";
+import { LoggingService } from "../../logging/logging.service";
+
+@Component({
+  selector: "app-user-list",
+  template: `
+    {{ authorNames }}
+    <span *ngIf="users.length > maxUserThreshold">
+      and {{ additionalUsers }} more
+    </span>
+  `,
+})
+export class UserListComponent implements OnInitDynamicComponent {
```

Comment:
There is already a component called UserList (the view displaying a table of all user accounts). This new component needs to be named differently